opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -981,9 +982,9 @@ Defaulted grid coordinates is not allowed for COMPDAT as part of ACTIONX)"
         bool update = false;
         if (well2.updateStatus(status)) {
             if (status == Well::Status::OPEN) {
-                auto new_rft = this->snapshots.back().rft_config().well_open(well_name);
+                auto new_rft = snapshot.rft_config().well_open(well_name);
                 if (new_rft.has_value())
-                    this->snapshots.back().rft_config.update( std::move(*new_rft) );
+                    snapshot.rft_config.update( std::move(*new_rft) );
             }
```

Comment:
Just to be sure .back() and [reportStep] is always the same here?

Reply:
And, within the function ,we have both usages, that is not good practice anyway.